src/Microsoft.Identity.Web.TokenAcquisition/AspNetCore/TokenAcquisition-AspnetCore.cs

```diff
@@ -20,7 +19,7 @@
 
 namespace Microsoft.Identity.Web
 {
-    internal class TokenAcquisitionAspNetCore : TokenAcquisition, ITokenAcquisitionInternal, ITokenAcquirerFactory
+    internal class TokenAcquisitionAspNetCore : TokenAcquisition, ITokenAcquisitionInternal
```

Comment:
Refactoring. the implementation of ITokenAcquirerFactory moved to the new DefaultTokenAcquirerFactory class